libs/design-system/src/icons/general/DelayAction.tsx

```diff
@@ -25,7 +24,7 @@ export function DelayAction(props: React.ComponentPropsWithoutRef<'svg'>) {
       </mask>
       <g mask="url(#mask0_433_2481)">
         <path
-          fill="url(#paint0_linear_433_2481)"
```

Comment:
Can you please make sure this doesn't remove the gradient on the step in the workflow editor?
<img width="382" alt="Screen Shot 2023-12-27 at 11 30 39" src="https://github.com/novuhq/novu/assets/7778680/772c5b1f-4906-4f27-aa77-e206f03da937">

Reply:
You are I used the incorrect icon by mistake! :(